user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Data/Spectrum_Buffer_Data.md

```diff
@@ -0,0 +1,9 @@
+---
+uid: Spectrum_Buffer_Data
+---
+
+# Spectrum buffer data
+
+The *Spectrum buffers* data category allows you to use data from [spectrum buffers](xref:Working_with_spectrum_monitors#watching-spectrum-buffers) in your DataMiner System as input for components in your dashboards or low-code apps.
+
+When you expand the *Spectrum buffers* section in the *Data* pane, you first need to select a Spectrum Analyzer element. Once selected, all available spectrum buffers are listed. To quickly find a specific spectrum buffer, use the filter box at the top of the data section.
```

Comment:
You can also drag the entire category/section as data and filter it using an element reference.